NewForce-at-Mountwest/bangazon-api-gnocci
# Completed Product Resource (ticket #2)

# Description

Created Git Method 
Created Get single to get Products by id.
Created Post method to create a new Product.
Created Put method to update a Product
Created a Soft Delete method that archives deleted item. Also created a Hard Delete method for testing purposes

## Related Ticket(s)
Ticket 2

## Steps to Test
Steps on how to run code:
Open GitBash
Clone the repo
cd bangazon-api-gnocci
cd bangazonApi
start bangazonApi.sln
--once the code is opened in Visual Studio 2019
F5 to run the code
https://localhost:5001/api/product <- to get all products
https://localhost:5001/api/product/1 <- to get single products

Run test unit to ensure working properly. 


# Checklist:
